packages/rust/compiler-rustc_driver-Cargo.toml.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/compiler/rustc_driver/Cargo.toml b/compiler/rustc_driver/Cargo.toml
+index 57159962a9e..6cba88226e3 100644
+--- a/compiler/rustc_driver/Cargo.toml
++++ b/compiler/rustc_driver/Cargo.toml
+@@ -41,3 +41,6 @@ winapi = { version = "0.3", features = ["consoleapi", "debugapi", "processenv"]
+ [features]
+ llvm = ['rustc_interface/llvm']
+ max_level_info = ['tracing/max_level_info']
++
++[target.armv7-linux-androideabi]
++ldflags = "-lgcc"
```

Comment:
Seems like you also need to add -lgcc_eh 

https://stackoverflow.com/questions/7885246/what-is-the-emutls-get-address-symbol

Reply:
No, that is for mac, there is no such lib in the NDK. The one that has it for armv7 in the NDK is `libgcc_real.a`, with `-lgcc` forwarding to that.

I'm not sure it's forwarding this linker flag, so having it blow up with fake flags now so I can check it went through.

Reply:
Yep, cargo throws out the `ldflags` I'm passing in, as I suspected:
```
warning: /home/builder/.termux-build/rust/src/compiler/rustc_driver/Cargo.toml: unused manifest key: target.armv7-linux-androideabi.ldflags
```